data/avro-schema/src/main/resources/avro/net/corda/data/preauth/PreAuthToken.avsc

```diff
@@ -0,0 +1,28 @@
+{
+  "type": "record",
+  "name": "PreAuthToken",
+  "namespace": "net.corda.data.membership.preauth",
+  "doc": "",
+  "fields": [
+    {
+      "name": "id",
+      "type": "string"
+    },
+    {
+      "name": "ownerX500Name",
+      "type": "string"
+    },
+    {
+      "name": "ttl",
+      "type": "long"
```

Comment:
I think the design specified that the input for TTL is a long but that should be stored and returned as an instant representing the instant when the TTL expires.